### PR TITLE
[CAR-29] Support specifying default open tab via ?section=overview|coverage|documents in member insurance details

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Support specifying default section in insurance details via URL

## Justify why they are needed

Part of Car Trial CRM flow, generally useful for link thab brings user close to insurance certificate

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3532** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3532" target="_blank">Graphite</a> 
    * **PR #3533** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3533" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3533?utm_source=stack-comment).

File: apps/store/src/utils/singleQueryParam.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { NextParsedUrlQuery } from 'next/dist/server/request-meta'
```
Comment (robinandeer) [inline apps/store/src/utils/singleQueryParam.ts:1]: is this the only place to import this type from?

**Review by robinandeer — APPROVED** [review]
Smart :)

Should we also update the URL when you click the different tabs? That way if the user reloads the page, they will see the tab they were viewing before

File: apps/store/src/utils/singleQueryParam.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/utils/singleQueryParam.ts:1]: Great question, replaced with `ParsedUrlQuery` from NodeJs standard lib

**Comment by alebedev** [comment]
### Merge activity

* **Nov 9, 8:30 AM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3533) rebased this pull request as part of a merge.

**Pull request merged**